### DISCUSSION TITLE
Talking Pictures TV

### DISCUSSION THREAD

**Discussion by what-a** [iptv-org/community#752] [Q&A]
### Channel Name

Talking Pictures TV

### Channel ID

TalkingPicturesTV.uk

### Country

UK

### Language

English

### Website

https://talkingpicturestv.co.uk/

### Stream URL

None that I've found. 😟

### Notes

I've never watched this channel but would like to see it. It seems like a UK version of Turner Classic Movies. Their website says "watch Talking Pictures TV on Virgin 445, Freesat 306, Freeview or Youview 81 or on the Sky digital satellite platform, channel 328."

**Comment by Sphinxroot** [comment]
Talking Pictures TV is the home of Vintage and Classic Programming targeting the over 65 demographic with a range of Films and Entertainment from the 1930's onwards.

only limited live stream
https://sat.tvmucho.com/app/tvmucho/live/talking-pictures

Talking Pictures TV now has a streaming movie service, TPTV Encore
https://www.tptvencore.co.uk/
VOD and free Register

https://talkingpicturestv.co.uk/films-and-shows/

User Comment:
Matthew
I've noticed Talking Pictures seem to endlessly repeat programmes, ( like all commercial stations these are paid for by advertising ) although I wonder if these particular productions are free of copyright. I quite enjoy the channel as there's so little else on offer

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19769
